ripuc.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+import scrapy
+
+
+class RipucSpider(scrapy.Spider):
+    name = 'ripuc'
+    allowed_domains = ['ripuc.org']
+    start_urls = ['http://www.ripuc.org/eventsactions/docket.html']
+
+    def parse(self, response):
+        table = response.xpath('//td[@class="normal"]/table')
+        number_of_rows = int(float(table.xpath('count(tr)').extract_first()))
+
+        for i in range(2, number_of_rows+1):
+            if table.xpath('tr[{}]/td[@rowspan="2"][1]/a/text()'.format(i)).extract():
```

Comment:
xpath 'tr[{}]/td[@rowspan="2"][1]/a/text()'.format(i)' is written twice.
Can we just execute it once and store in variable, and then check it and use it.
Same is repeated below in the spider as well.